packages/react-reconciler/src/ReactStrictModeWarnings.js

```diff
@@ -141,12 +141,18 @@ if (__DEV__) {
 
       lowPriorityWarning(
         false,
-        'componentWillMount is deprecated and will be removed in the next major version. ' +
-          'Use componentDidMount instead. As a temporary workaround, ' +
-          'you can rename to UNSAFE_componentWillMount.' +
-          '\n\nPlease update the following components: %s' +
-          '\n\nLearn more about this warning here:' +
-          '\nhttps://fb.me/react-async-component-lifecycle-hooks',
+        'componentWillMount has been renamed to UNSAFE_componentWillMount, ' +
+          "and the old name won't work in the next major version of React.\n" +
+          'We suggest doing one of the following:\n' +
+          '- If you initialize state in componentWillMount, move this logic into the constructor.\n' +
+          '- If you fetch data or perform other side effects in componentWillMount, ' +
+          'move this logic into componentDidMount.\n' +
+          '- To rename all deprecated lifecycles to their new names, you can run ' +
```

Comment:
Maybe we should give a hint as to why folks might not want to just rename and ignore? Maybe we could replace

"_To rename all deprecated lifecycles to their new names, ..._"

...with

"_To prevent this warning from being logged to the console, ..._"

...so it sounds a little more like you're just choosing to ignore the warning without fixing the problem?

(here and below)

Reply:
How about "To ignore this warning from being logged to the console..."

Reply:
rather, "to disable this warning..."

Reply:
It's so wordy :(

Reply:
I think that we're saying the warning will still log in strict mode, and that we've moved this to the bottom of the list, is sufficient. I'll fight you on this briannson.